dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/test/java/org/apache/dolphinscheduler/plugin/task/api/utils/ProcessUtilsTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.task.api.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ProcessUtilsTest {
+
+    private static final Pattern LINUXPATTERN = Pattern.compile("\\((\\d+)\\)");
+
+    private String getPidStr(String pids) {
+        Matcher mat = null;
+        StringBuilder sb = new StringBuilder();
+        if (StringUtils.isNotEmpty(pids)) {
+            mat = LINUXPATTERN.matcher(pids);
+        }
+        if (null != mat) {
+            while (mat.find()) {
+                sb.append(mat.group(1)).append(" ");
+            }
+        }
+        return sb.toString().trim();
+    }
```

Comment:
Please directly use method in `ProcessUtils`.

Reply:
> Please directly use method in `ProcessUtils`.

![image](https://github.com/apache/dolphinscheduler/assets/42668765/c51a429a-e447-42c6-a31f-2108d5cb8897)
Can the method "getPidsStr" run right like the screenshot? (Because there is no processes in UT)

Reply:
You can mock the return value of `OSUtils.exeCmd`

Reply:
> You can mock the return value of `OSUtils.exeCmd`, otherwise, this UT is meaningless without any code coverage.
![image](https://github.com/apache/dolphinscheduler/assets/42668765/a9ad3b9c-d2ea-4334-a1d7-4d9f096ced9d)

If i mock the return value of `OSUtils.exeCmd`, my method is to copy the code from `getPidsStr` from `ProcessUtils`.
Or there is a better way to do that in UT?

Reply:
You can refer to other test files, and use Mockito API.

Reply:
> You can refer to other test files, and use Mockito API.

Please take a look, thanks

Reply:
Please check failed UT.

Reply:
> Please check failed UT.

Ok

Reply:
Any other problems? 😉